SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+The NEST Desktop is research oriented software and as such is expected to
+run mainly in well protected environments. In case it is found that this
+software can be used to violate security mechanisms, developers will try to
+provide patches that mitigate the risk of this misuse.
```

Comment:
```suggestion
provide patches that mitigate the risk of this misuse. However, this
cannot be guaranteed and usually, this might be realized only for the most
recent version available. Please check the repository for available
updates.
```